apps/libsignpost/port_signpost_tock.c

```diff
@@ -0,0 +1,140 @@
+#include <stdint.h>
+#include <string.h>
+#include "tock.h"
+#include "port_signpost.h"
+#include "i2c_master_slave.h"
+#include "gpio.h"
+#include "led.h"
+#include "timer.h"
```

Comment:
style nit: one line between system and local headers, alpha order the local headers